README.md

```diff
@@ -171,198 +75,11 @@ Unleash Edge is distributed as a binary and as a docker image.
-          How long between each refresh for a token [env: FEATURES_REFRESH_INTERVAL_SECONDS=] [default: 10]
-      --token-revalidation-interval-seconds <TOKEN_REVALIDATION_INTERVAL_SECONDS>
-          How long between each revalidation of a token [env: TOKEN_REVALIDATION_INTERVAL_SECONDS=] [default: 3600]
-  -t, --tokens <TOKENS>
-          Get data for these client tokens at startup. Accepts comma-separated list of tokens. Hot starts your feature cache [env: TOKENS=]
-  -H, --custom-client-headers <CUSTOM_CLIENT_HEADERS>
-          Expects curl header format (-H <HEADERNAME>: <HEADERVALUE>) for instance `-H X-Api-Key: mysecretapikey` [env: CUSTOM_CLIENT_HEADERS=]
-  -s, --skip-ssl-verification
-          If set to true, we will skip SSL verification when connecting to the upstream Unleash server [env: SKIP_SSL_VERIFICATION=]
-      --pkcs8-client-certificate-file <PKCS8_CLIENT_CERTIFICATE_FILE>
-          Client certificate chain in PEM encoded X509 format with the leaf certificate first. The certificate chain should contain any intermediate certificates that should be sent to clients to allow them to build a chain to a trusted root [env: PKCS8_CLIENT_CERTIFICATE_FILE=]
-      --pkcs8-client-key-file <PKCS8_CLIENT_KEY_FILE>
-          Client key is a PEM encoded PKCS#8 formatted private key for the leaf certificate [env: PKCS8_CLIENT_KEY_FILE=]
-      --pkcs12-identity-file <PKCS12_IDENTITY_FILE>
-          Identity file in pkcs12 format. Typically this file has a pfx extension [env: PKCS12_IDENTITY_FILE=]
-      --pkcs12-passphrase <PKCS12_PASSPHRASE>
-          Passphrase used to unlock the pkcs12 file [env: PKCS12_PASSPHRASE=]
-      --upstream-certificate-file <UPSTREAM_CERTIFICATE_FILE>
-          Extra certificate passed to the client for building its trust chain. Needs to be in PEM format (crt or pem extensions usually are) [env: UPSTREAM_CERTIFICATE_FILE=]
-
-  -h, --help
-          Print help
-
-```
-
-#### Offline
-
-```mermaid
-graph LR
-  A(Client) -->|Fetch toggles| B((Edge))
-  B-->|Fetch toggles| C[Features dump]
-```
-
-Offline mode is useful when there is no connection to an upstream node, such as your Unleash instance or another Edge
-instance, or as a tool to make working with Unleash easier during development.
-
-To use offline mode, you'll need a features file. The easiest way to get one is to download a JSON dump of a result from
-a query against an Unleash server on
-the [/api/client/features](https://docs.getunleash.io/reference/api/unleash/get-client-feature) endpoint. You can also
-use a hand rolled, human readable JSON version of the features file. Edge will automatically convert it to the API
-format when it starts up. Here's an example:
-
-``` json
-{
-  "featureOne": {
-    "enabled": true,
-    "variant": "variantOne"
-  },
-  "featureTwo": {
-    "enabled": false,
-    "variant": "variantTwo"
-  },
-  "featureThree": {
-    "enabled": true
-  }
-}
-```
+## Concepts (see [Concepts](./docs/concepts.md))
```

Comment:
Given what we did for deployment above, maybe it would be more consistent as this:

```suggestion
## Concepts

See our page on [Edge concepts](./docs/concepts.md)
```